pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-spark/pom.xml

```diff
@@ -36,6 +36,7 @@
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
     <phase.prop>package</phase.prop>
+    <scala.binary.version>2.11</scala.binary.version>
```

Comment:
this is already declaired in main pom.xml? or were you also meant to change the main pom.xml scala.binary.version to 2.13 as well?

Reply:
spark has no scala 2.13 support until 3.2.x， ref: https://mvnrepository.com/artifact/org.apache.spark/spark-core.

This is to keep the same build behavior.

Reply:
oh I meant why do we need this since the main pom already have the identical setting

Reply:
ah, got it, since the spark is only used in the plugin repo, I feel we should just move it from root pom to the plugin level similar to Kafka, so later on we can also add spark 3.x support